master/internal/db/setup.go

```diff
@@ -80,6 +112,11 @@ func Connect(opts *config.DBConfig) (*PgDB, error) {
 
 	db.sql.SetMaxOpenConns(maxOpenConns)
 
+	err = checkPostgresVersion(db)
+	if err != nil {
+		return nil, fmt.Errorf("error checking Postgres version: %s", err)
```

Comment:
I'm open to being convinced, but I think we want to log this error ("failed to check Postgres version" or something) and return a non-nil error here. I'd hate for somebody's cluster to not start up correctly purely because we couldn't warn about an out-of-date Postgres.

Reply:
That's not a bad point, I'll make this swap; it's better to just log this and not prevent start up